js/foundation.toggler.js

```diff
@@ -65,6 +65,21 @@ class Toggler extends Plugin {
         'aria-controls': id,
```

Comment:
There is still the old logic left, `var id` and others are set multiple times which can cause issues.
It might make more sense to create new variables in the new code block or extend the old logic with the new one.

Reply:
Quite right! Missed that one. I'll have another look to make sure that I haven't missed anything outside of the changed code. PR coming soon.